script/verify.py

```diff
@@ -81,36 +82,8 @@ async def verify(self, ctx, name : str = "", entry_number : str = ""):
         if(check_entry_number(entry_number) == False):
             return await ctx.send("Invalid Entry Number")
         email = entry_number + "@iitjammu.ac.in"
-        verf_msg = f"An email with Verification Code has been sent to {email}. Enter your code here within {timeout} minutes."
-        code = random_with_N_digits(6)
-        
-        print(ctx.author, "verification Started")
-        await logs.print(f'{ctx.author.mention} started Verification')
 
-        message = Mail(
-            from_email='Ava-noreply@'+os.getenv('EMAIL_DOMAIN'),
-            to_emails=email,
-            subject='Coding Club Discord Server',
-            html_content= f'Thanks {entry_number} for creating account on Coding Club Discord Server,<br>Your Verification Code is : <b>{code}</b><br>This Code will Expire in 3 Minutes.<br><br>Sincerely,<br>Ava, BOT Coding Club'
-            )
-        try:
-            if(DEBUG == False):
-                sg = SendGridAPIClient(os.getenv('SENDGRID_API_KEY'))
-                response = sg.send(message)
-                print(response.status_code)
-            else:
-                print(message)
-        except Exception as e:
-            print(e.message)
-        await ctx.send(f'Name : {name}\nEntry Number : {entry_number}\n{verf_msg}')
-        try:
-            code_got = await self.bot.wait_for('message', check=lambda message: message.author == ctx.author, timeout = 180)
-        except asyncio.TimeoutError:
-            await ctx.send(f"{ctx.author} Verification Time Out")
-            return await logs.print(f'{ctx.author.mention} verification Timeout')
-        if(code_got.content[0] == "."):
-            return
-        if(code_got.content == str(code)):
+        if(self.authorize_user(ctx, logs, name, entry_number)):
```

Comment:
add await with == true, so it will not bypass this in exception

Reply:
done